tests/fixtures/ddevProj1/.ddev/config.yaml

```diff
@@ -1,67 +1,96 @@
 name: setup-ddev-proj1
 type: php
 docroot: ""
-php_version: "8.0"
+php_version: "8.3"
 webserver_type: nginx-fpm
 router_http_port: "80"
 router_https_port: "443"
 xdebug_enabled: false
 additional_hostnames: []
 additional_fqdns: []
-mariadb_version: "10.2"
-provider: default
+database:
+    type: mariadb
+    version: "10.2"
```

Comment:
Next time around I think this would be better as the default (leave it out or specify 10.11)